src/components/Toggle.tsx

```diff
@@ -60,21 +42,40 @@ export const Toggle = (props: Props) => {
             ...(props.value ? getInlineStyle('circleActive') : {}),
           }}
         >
-          <button
+          <input
```

Comment:
I am not sure anymore, but I think this one caused some issues (maybe on mobile devices). That is why we did not change it to input so far. Please test very carefully otherwise Björn is gonna burn 😜